tests/pubsub/testfloodsub.nim

```diff
@@ -60,7 +60,7 @@ suite "FloodSub":
       await nodes[1].subscribe("foobar", handler)
       await waitSub(nodes[0], nodes[1], "foobar")
 
-      await nodes[0].publish("foobar", "Hello!".toBytes())
+      discard await nodes[0].publish("foobar", "Hello!".toBytes())
```

Comment:
`check: await... == 1` (or whatever it should be?)

Reply:
`check: 1 == await ... ` maybe easier to read

Reply:
that's from my branch btw 😄 , the thing is that this does not apply to flood, it will always be 0.

Reply:
btw I find it useful (nbc side for example I'm using it) to have this int with gossip. but we can remove it once things stabilize if you guys don't think it's necessary.. it's a debug tool tho